test/Report.php

```diff
@@ -229,6 +230,11 @@ public function render($template, $data = array()) {
 		$template = Libraries::locate('test.templates', $template, array(
 			'filter' => false, 'type' => 'file', 'suffix' => ".{$config['format']}.php"
 		));
+
+		if($template === null) {
```

Comment:
Between the `if` and the `(` one whitespace is needed to be in line with the CS.